src/main/java/org/spongepowered/api/command/args/CommandContext.java

```diff
@@ -116,6 +125,53 @@ public CommandContext() {
         return getOne(ArgUtils.textToArgKey(key));
     }
 
+    /**
+     * Gets the value for the given key if the key has only one value, throws an
+     * exception otherwise.
+     *
+     * @param key the key to get
+     * @param <T> the expected type of the argument
+     * @return the argument
+     * @throws java.util.NoSuchElementException if there is no element with the
+     *      associated key
+     * @throws IllegalArgumentException if there are more than one element
+     *      associated with the key (thus, the argument is illegal in this
+     *      context)
+     * @throws ClassCastException if the element type is not what is expected
+     *      by the caller
+     */
+    @SuppressWarnings("unchecked")
+    public <T> T getOneUnchecked(String key)
```

Comment:
Should be `requireOne` to match the `require` name used in the Data API: https://github.com/SpongePowered/SpongeAPI/blob/bleeding/src/main/java/org/spongepowered/api/data/value/mutable/CompositeValueStore.java#L106

Reply:
I have no issue with that. For reference, note that I got this name from the ServiceManager methods, `provide` and `provideUnchecked`, in case that wants to be rationalised later to `get` and `require`.